setup.py

```diff
@@ -23,25 +23,26 @@
     ]
 
 api_requirements = [
-    'markupsafe == 2.0.1',  # While flask revision < 2
-    'flask == 1.1.4',
+    'markupsafe >= 2.0.1, < 2.1.0',
+    'flask >= 1.1.4, < 2.0.0',
     'flask-cors == 3.0.10',
     'gunicorn >= 20.0.0, < 21.0.0',
-    'werkzeug >= 1.0.0, < 2.0.0',
+    'werkzeug >= 1.0.1, < 2.0.0',
     ]
 
 dev_requirements = [
     'autopep8 >= 1.4.0, < 1.6.0',
     'coverage == 6.0.2',
     'darglint == 1.8.0',
-    'flake8 >= 4.0.0, < 4.1.0',
+    'flake8 >= 3.9.0, < 4.0.0',
```

Comment:
I am not sure I understand: this reverts #1137. Are you saying that the problem documented in #1136 was not coming from this dependency? Yet we saw that it fixed it. Downgrading `flake8` and forcing it to v3 does not seem like a sustainable solution.

Reply:
Upgrading flake8 introduced a dependency conflict with sphinx, that was being silenced by `test-doc`.